### PR TITLE
Bgfix/change structure

### PR DESCRIPTION
**Pull request by DavidKizinger**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
 import { Switch } from "react-router-dom";
 import Router from "./Routes/Router";
-import "./Style/index.scss";
+import NavBar from "./Components/NavBar";
+import Header from "./Components/Header";
+import { useStyles } from "./Containers/MainPage/useStyles";
+
+export default function App() {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
```
Comment (Sahakgm) [inline src/App.js:10]: useState import ara verevum

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,24 @@
-import React from "react";
```
Comment (Sahakgm) [inline src/App.js:1]: bolor texer@ nuyn style-ov gri ("" or '')

File: src/Components/NavBar/First/index.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import MoveToInboxIcon from "@material-ui/icons/MoveToInbox";
+import { ListItem, ListItemIcon, ListItemText } from "@material-ui/core";
+import { NavLink } from "react-router-dom";
+import { useStyles } from "../../../Containers/MainPage/useStyles";
+
+const MainListItems = () => {
+  const styles = useStyles()
+  return (
+    <div>
+      <NavLink to="/" className={styles.link}>
```
Comment (Sahakgm) [inline src/Components/NavBar/First/index.jsx:14]: porci dynamic sarqes u map-pv render anes

File: src/Components/UsersList/index.jsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+
+function createData(name, calories, fat, carbs, protein) {
+  return { name, calories, fat, carbs, protein };
+}
+
+const UsersList = () => {
+  const DATE = new Date();
+  const date = `${DATE.getDay()}.${DATE.getMonth()}.${DATE.getFullYear()}`;
```
Comment (Sahakgm) [inline src/Components/UsersList/index.jsx:16]: karas moment.js ogtagorces

File: src/Containers/MainPage/index.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import UsersList from "../../Components/UsersList";
+
+const MainPage = () => {
```
Comment (Sahakgm) [inline src/Containers/MainPage/index.jsx:4]: es component-um urish banela linelu te menak UsersList -@?

Comment (DavidKizinger) [reply src/Containers/MainPage/index.jsx:4]: Der ayo,menak es

**Pull request merged**